bats_ai/core/models/recording.py

```diff
@@ -10,6 +10,19 @@
 logger = logging.getLogger(__name__)
 
 
+class RecordingTag(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    text = models.CharField(max_length=50)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=['user', 'text'], name='unique_user_text_tag')
```

Comment:
So right now user's can't delete their own tags within the system (they can remove from recordings).  Should tags be unique per user or should be unique for the whole system? 

I'm wondering if we should have the tags global, that way when someone is adding tags to a file they can see the other existing tags and add the same tag do prevent duplicated tags and make filtering for public data a bit easier?

Maybe leave it as is for now while I do some more thinking.

Reply:
I don't have strong feelings either way, and I suppose it depends on how people end up using the tagging system. If each user has their own approach to tags then being exposed to everyone's tags when creating/editing a recording seems more annoying than helpful (maybe my tags are only meaningful to me).